Source/OpalGraphics/cairo/CairoFontWin32.m

```diff
@@ -260,8 +263,8 @@ - (CGGlyph) glyphWithCharacter: (unichar)character
 }
 
 - (bool) getGlyphAdvances: (const CGGlyph[])glyphs
-                         : (size_t)count
-                         : (int[]) advances
+: (size_t)count
+: (int[]) advances
 {

```

Comment:
This  new indentation doesn't seem right and even the fix is not consistent:

```
: (p)q
: (a)b
: (c) d
```

There additional cases like this one.